ynr/apps/elections/templates/elections/sopn_for_ballot.html

```diff
@@ -40,10 +40,10 @@ <h2>
       </p>
     {% endif %}
 
+    {% include "elections/includes/_sopn_debug.html" %}
```

Comment:
Oh sorry, can you wrap this in a `if request.users.is_authenticated`?

Reply:
Done! I'll fix up the commits now